pkg/resource/aws/aws_route53_health_check.go

```diff
@@ -41,3 +44,9 @@ func (r *AwsRoute53HealthCheck) TerraformType() string {
 func (r *AwsRoute53HealthCheck) CtyValue() *cty.Value {
 	return r.CtyVal
 }
+
+func initAwsRoute53HealthCheckMetaData(resourceSchemaRepository resource.SchemaRepositoryInterface) {
+	resourceSchemaRepository.SetNormalizeFunc(AwsRoute53HealthCheckResourceType, func(val *resource.Attributes) {
+
+	})
```

Comment:
Why is this empty ? And if it is intentional, why ?

Reply:
I need to remove it nice catch thanks